typescript/vrotest/package.json

```diff
@@ -49,15 +50,20 @@
     "minimist": "1.2.6",
     "rimraf": "3.0.2",
     "rollup": "3.29.5",
-    "@rollup/plugin-terser": "0.4.4",
+    "source-map": "^0.8.0-beta.0",
     "ts-node": "10.5.0",
-    "typescript": "4.5.5",
-    "source-map": "^0.8.0-beta.0"
+    "typescript": "4.5.5"
   },
   "bundledDependencies": [
     "iconv-lite",
     "jasmine",
     "nyc",
     "browserslist"
+  ],
+  "bundleDependencies": [
```

Comment:
What's the diff here between `bundledDependencies` and `bundleDependencies` syntax? are they used for different purposes?

Reply:
Interesting, I haven't really changed this manually. It should have changed when I have ran an NPM command at some point.
From what I see in the documentation the formal name of the property is "[bundleDependencies](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bundledependencies)", as there is a remark stating:
_If this is spelled "bundledDependencies", then that is also honored._
Other than that I don't think there are functional differences. Will keep only the formal name of the property.

Reply:
Hmmm now that you mention this, I think I saw something similar. I guess both are honored but if the default one is missing, it gets generated. Oks, thanks!